cypress/support/pages/teams/pageCommonTeams.js

```diff
@@ -20,110 +19,129 @@ class Teams_Common {
   static #selectInternalTeamMember = '[data-testid="select_team_members_add"]'
   static #studentTableBody = '[data-testid="students_names_container"]'
   static #confirmTeamMemberAddButton = '[data-testid="btn-submit"]'
-  static #confirmTeamMemberDeleteButton= '[data-testid="btn-submit"]'
+  static #confirmTeamMemberDeleteButton = '[data-testid="btn-submit"]'
   static #deleteIconInTableViewRow = '[data-testid="btn-delete-team-member"]'
 
-
-  doNotSeeDeletedStudentInTeam(){
-    cy.get(Teams_Common.#studentTableBody).find('tr td','Herbert').should('not.exist')
+  doNotSeeDeletedStudentInTeam () {
+    cy.get(Teams_Common.#studentTableBody)
+      .find('tr td', 'Herbert')
```

Comment:
method should be called with param studentName instead of a hard coded value ("Herbert") in it.